canonical-web-and-design/microk8s.io
# Build high availability page

## Done

- Build HA page at `/high-availability`
- Add side nav
- Add `Features` and `Community` to main nav
- Remove `Code` and `Discourse` from main nav

**Known differences** : 
- `Tutorials` not added to main nav yet as page hasn't been built
- No `Overview` link in side nav as landing features page no longer being built
- Raft algo animation on `/high-availability` left out for now


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/high-availablity
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page matches [copy doc](https://docs.google.com/document/d/1G6FijpahZCSfxcuFAkv22wKCILEkF2u4Q0TQUfkW5i4/edit#)
- Check [Menu](https://docs.google.com/document/d/14TiB5SUGYbjVHRycUC5iS7c4J-3vW7kbXCO_TuB6xO0/edit) and side nav


## Issue / Card

Fixes [#3263](https://github.com/canonical-web-and-design/web-squad/issues/3263)
